src/cascadia/TerminalApp/App.xaml

```diff
@@ -77,8 +77,16 @@
 
                         <ResourceDictionary x:Key="Light">
                             <!--  Define resources for Light mode here  -->
+
+                            <!--
+                                The Dark mode BG uses

```

Comment:
are we not also changing the contrast in dark mode? i thought that they had new values for that one as well..

Reply:
I just looked through the thread again and I can't find anything about changing the dark mode colors. The contrast there is a lot less egregious, so that I'm okay with.